xfields/beam_elements/beambeam3d.py

```diff
@@ -99,7 +99,7 @@ class BeamBeamBiGaussian3D(xt.BeamElement):
     }
 
     def __init__(self,
-                    phi=None, alpha=None, other_beam_q0=None,
+                    phi=None, alpha=None, other_beam_q0=None, particles_per_macroparticle = None,
```

Comment:
Particles has the attribute ```weight```, can it be used instead of ```particles_per_macroparticle```?

Reply:
I merge it as it is for now, and add a comment in the [issue](https://github.com/xsuite/xsuite/issues/125#issuecomment-1250019669)